CMakeCache.txt

```diff
@@ -0,0 +1,356 @@
+# This is the CMakeCache file.
+# For build in directory: /home/liyan/vml/OpenVML
```

Comment:
Isn't this file a CMake cache file and will be regenerated every time during configuration? Maybe it's a mistake to add these files.